packages/sdk/src/_internal/Extensions.ts

```diff
@@ -31,3 +31,42 @@ export function callExtension(args: CallOnExtensionParams): PopulatedExtensionCa
     address: args.comptrollerProxy,
   });
 }
+
+export function callOnExtensionDecode(data: Hex) {
```

Comment:
All other encode/decode functions that we use in the SDK are doing parameter encoding/decoding and not function encoding/decoding. I therefore think it's confusing if we do function decoding here, and I suggest we remove this and call this function directly in the backend (like we currently do in the frontend).